upgrade/v14xto150/upgrade.go

```diff
@@ -21,6 +22,10 @@ const (
 )
 
 func UpgradeResources(namespace string, lhClient *lhclientset.Clientset, kubeClient *clientset.Clientset, resourceMaps map[string]interface{}) error {
+	if err := upgradeCSIPlugin(namespace, kubeClient); err != nil {
```

Comment:
Why we need to do this specially? isn't it handled by driver deployment instead?

Reply:
You can check [item 2 in this comment](https://github.com/longhorn/longhorn/issues/6139#issuecomment-1596457985) for details